services/mytest.go

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright The RAI Inc.
+ * The RAI Authors
+ */
+
+package services
+
+import (
+	"bean/repositories"
+
+	"github.com/labstack/echo/v4"
+)
+
+type MyTestService interface {
+	GetMasterSQLTableList(c echo.Context) (map[string]interface{}, error)
+}
+
+type myTestService struct {
+	myTestRepository repositories.MyTestRepository
+}
+
+func NewMyTestService(myTestRepo repositories.MyTestRepository) *myTestService {
```

Comment:
should return interface MyTestService

Reply:
we should return struct instead of interface
https://medium.com/@cep21/what-accept-interfaces-return-structs-means-in-go-2fe879e25ee8

Reply:
ok